python/coqtail.py

```diff
@@ -337,6 +337,7 @@ def send_until_fail(
         opts: VimOptions,
     ) -> Tuple[Optional[Tuple[int, int]], Optional[str]]:
         """Send all sentences in 'send_queue' until an error is encountered."""
+        empty = self.send_queue == deque()
```

Comment:
`len(self.send_queue) == 0`?

Reply:
The PEP 8 recommendation is to use a sequence's implicit booleanness to check if it's empty, but I've never liked that. I expect in this case there's no significant performance difference between using `len`, or `bool`, or comparing to `deque`, but I just like the explicitness of the 3rd option since it makes it clear what the type is. I really wish there was just a `.is_empty()` method.